### PR TITLE
Add tutorial on using Orbbec 3D cameras (UVC)

### PR DESCRIPTION
**Pull request by Andy-SAKN**
### Pull Request Readiness Checklist

See details at https://github.com/opencv/opencv/wiki/How_to_contribute#making-a-good-pull-request

- [x] I agree to contribute to the project under Apache 2 License.
- [x] To the best of my knowledge, the proposed patch is not based on a code under GPL or another license that is incompatible with OpenCV
- [x] The PR is proposed to the proper branch
- [ ] There is a reference to the original bug report and related work
- [ ] There is accuracy test, performance test and test data in opencv_extra repository, if applicable
      Patch to opencv_extra has the same branch name.
- [ ] The feature is well documented and sample code can be built with the project CMake

### CODE REVIEW

File: doc/tutorials/app/orbbec_uvc.markdown [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+
+
+
+
+### Note
+
+1. If the camera fails to open, please first check the camera permissions under Privacy and Security settings. If an adapter is used, ensure the related drivers are functioning correctly.
+2. Do not run the code simultaneously with Orbbec's proprietary software, as this will result in resource conflicts with the camera.
+3. If the code does not run successfully, try restarting your device.
+4. OpenCV version 4.10 or newer is required.
```
Comment (asmorkalov) [inline doc/tutorials/app/orbbec_uvc.markdown:142]: Please add table with author and compatibility notes like here: https://github.com/opencv/opencv/blob/4.x/doc/tutorials/calib3d/camera_calibration/camera_calibration.markdown

File: doc/tutorials/app/orbbec_uvc.markdown [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+
+
+### Introduction
+
+This tutorial is devoted to the Orbbec 3D cameras based on UVC protocol. For the use of the older Orbbec 3D which depends on OpenNI, please refer to the [previous tutorial](https://github.com/opencv/opencv/blob/4.x/doc/tutorials/app/orbbec_astra_openni.markdown).
+
+Unlike working with the OpenNI based Astra 3D cameras which requires OpenCV built with OpenNI2 SDK, Orbbec SDK is not required to be installed for accessing Orbbec UVC 3D cameras via OpenCV. By using `cv::VideoCapture` class, users get the stream data from 3D cameras, similar to working with USB cameras. The calibration and alignment of the depth map and color image are done internally.
+
+
+
```
Comment (asmorkalov) [inline doc/tutorials/app/orbbec_uvc.markdown:17]: Please remove extra empty lines here and below in the document.

Comment (Andy-SAKN) [reply doc/tutorials/app/orbbec_uvc.markdown]: Thanks! I have fixed the two problems you mentioned above.

File: doc/tutorials/app/orbbec_uvc.markdown (hunk 1 above)
Comment (asmorkalov) [reply doc/tutorials/app/orbbec_uvc.markdown:120]: You can remove the 4th item now.

File: doc/tutorials/app/orbbec_uvc.markdown [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+  After exiting the loop, the camera resources are released.
+### Results
+
+#### Python
+
+![RGB And DEPTH frame](images/orbbec_uvc_python.jpg)
+
+#### C++
+
+![RGB And DEPTH And DepthToColor frame](images/orbbec_uvc_cpp.jpg)
```
Comment (kaingwade) [inline doc/tutorials/app/orbbec_uvc.markdown:113]: Can you add toggle to show the two pictures, same as the switch to show the two code version?

File: doc/tutorials/app/orbbec_uvc.markdown (hunk 1 above)
Comment (Andy-SAKN) [reply doc/tutorials/app/orbbec_uvc.markdown]: Thanks! It has been removed now.

File: doc/tutorials/app/orbbec_uvc.markdown [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+
+### Note
+
+1. If the camera fails to open, please first check the camera permissions under Privacy and Security settings. If an adapter is used, ensure the related drivers are functioning correctly.
+2. Do not run the code simultaneously with Orbbec's proprietary software, as this will result in resource conflicts with the camera.
+3. If the code does not run successfully, try restarting your device.
```
Comment (kaingwade) [inline doc/tutorials/app/orbbec_uvc.markdown:119]: Can be removed too in my opinion.

File: doc/tutorials/app/orbbec_uvc.markdown [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+
+- **Loop to Grab and Process Data**:
+  In an infinite loop, the code continuously grabs data from the camera. The `obsensorCapture.grab()` method attempts to grab a frame.
+
+- **Process RGB Image**:
+  Using `obsensorCapture.retrieve(image, CAP_OBSENSOR_BGR_IMAGE)` to retrieve the RGB image data. If successfully retrieved, the RGB image is displayed in a window using `imshow("RGB", image)`.
+
+- **Process Depth Image**:
+  Using `obsensorCapture.retrieve(depthMap, CAP_OBSENSOR_DEPTH_MAP)` to retrieve the depth image data. If successfully retrieved, the depth image is normalized and a false color image is applied, then the result is displayed in a window using `imshow("DEPTH", adjDepthMap)`. The retrieved depth values are in millimeters and are truncated to a range between 300 and 5000 (millimeter).
+  This fixed range can be interpreted as a truncation based on the depth camera's measurement limits, potentially enhancing the visualization's relevance to a specific depth range.
```
Comment (kaingwade) [inline doc/tutorials/app/orbbec_uvc.markdown:95]: measurement limits --> depth range; potentially enhancing the visualization's relevance to a specific depth range --> removing invalid pixels on the depth map.

Comment (Andy-SAKN) [reply doc/tutorials/app/orbbec_uvc.markdown]: Thanks a lot! I have squashed the commites into one now.

**Comment by kaingwade** [comment]
@Andy-SAKN CI reports:

> doc/tutorials/app/orbbec_uvc.markdown:1: trailing whitespace.
+Using Orbbec 3D cameras (UVC) {#tutorial_orbbec_uvc} 
doc/tutorials/app/orbbec_uvc.markdown:29: trailing whitespace.
+----

**Review by kaingwade — APPROVED** [review]
LGTM.

**Review by asmorkalov — APPROVED** [review]

**Pull request merged**